### ISSUE TITLE
feature request: write id3v2 instead of APEv2

### ISSUE THREAD

**Issue by E-t-z** [Sappharad/mp3gainOSX#34]
It is possible to do it via command line using "-s i" switch, but not via GUI.
Maybe this option could be added as on some devices handling APEv2 can be problematic and can cause playback issues.

Thanks.

**Comment by Sappharad** [comment]
This sounds like a good idea and should be fairly easy to add in a small update.

**Comment by Sappharad** [comment]
I received an email today requesting /s s
Perhaps a drop-down to pick any of the /s options.

**Comment by E-t-z** [comment]
Makes sense, especially since you don't have much motivation to improve this app as you no longer use it yourself.
